### DISCUSSION TITLE
Address sanitizer disabling

### DISCUSSION THREAD

**Discussion by argirios44** [SeisSol/SeisSol#690] [Q&A] [answered]
How can i disable address sanitizer errors? I have been waiting 5 hours for the tpv33 simulation to complete only for it to get aborted because it could not dealocate memory, that was painful. Not sure where i have to put this flag.
![image](https://user-images.githubusercontent.com/35249593/194727658-fb5d6837-149a-4666-b928-d61e0d399a52.png)

**Comment by Thomas-Ulrich** [comment, marked as answer]
Run ccmake in seissol build folder and turn off the option with sanitizer? 

Le 8 octobre 2022 23:01:24 GMT+02:00, Argyrios Gklaras ***@***.***> a écrit :

**Reply by argirios44** [reply]
That worked, thanks!
